supabase/config.toml

```diff
@@ -0,0 +1,202 @@
+# A string used to distinguish different Supabase projects on the same host. Defaults to the
+# working directory name when running `supabase init`.
+project_id = "commercetest-nlnet"
+
+[api]
+enabled = true
+# Port to use for the API URL.
+port = 54321
+# Schemas to expose in your API. Tables, views and stored procedures in this schema will get API
+# endpoints. `public` is always included.
+schemas = ["public", "graphql_public"]
+# Extra schemas to add to the search_path of every request. `public` is always included.
+extra_search_path = ["public", "extensions"]
+# The maximum number of rows returns from a view, table, or stored procedure. Limits payload size
+# for accidental or malicious requests.
+max_rows = 1000
```

Comment:
I'm, curious whether we'd have queries that may return more than `max_rows`... something to be aware of in case it bites us in the data science work.

Reply:
Exactly. We might need to consider the paid version of it.

Reply:
Ah, OK, I didn't realise that the limit was set for the free version of supabase. Good to know. Thank you. If this is the case, please can you update the comment in lines 14&15 to say so; otherwise the comment is potentially misleading.